test/test_parser.py

```diff
@@ -1,14 +1,13 @@
 import unittest
-
-import pyglet
+import os
 
 import pywavefront.parser
 
 class TestParsers(unittest.TestCase):
     def setUp(self):
-        pyglet.resource.path.append('@' + __name__)
-        pyglet.resource.reindex()
-        meshes = pywavefront.Wavefront('simple.obj')
+        # Append current path to locate files
+        folder = os.path.dirname(__file__) + '/'
+        meshes = pywavefront.Wavefront(folder + 'simple.obj')
```

Comment:
Example for this would be `os.path.join(os.path.dirname(__file__), 'simple.obj')`